### ISSUE TITLE
Is it possible to convert xls(x) files to pdf completely even if some cells are shrink and hidden by adjacent cells?

### ISSUE THREAD

**Issue by koreaidi0t** [unoconv/unoconv#589]
When I try to convert xls(x) files that have some shrink cells to pdf files, I can get pdfs not containing all value of the cells.

Is it possible to get a pdf containing whole value of cells from a xls(x)?

I used the following command.
`unoconv -f pdf test.xlsx`

input xlsx file
![image](https://user-images.githubusercontent.com/26409122/145074659-159e94bf-5f87-4d24-bd1b-6d808475d240.png)

ouput pdf file
![image](https://user-images.githubusercontent.com/26409122/145074821-bece16ee-1cb9-49e5-bb30-196e24a045a6.png)

Thanks

**Comment by faisalraza-pixonomi** [comment]
@koreaidi0t Did you find anything regrading this? It seems like `unoconv` don't have support as [it is deprecated](https://github.com/unoconv/unoconv?tab=readme-ov-file#unoconv-is-deprecated).

**Comment by regebro** [comment]
No, converting to PDF is essentially like printing, but to a file. If the value is hidden, it is hidden.
